Makefile

```diff
@@ -115,7 +115,9 @@ else
 endif
 	# This might help with emfile errors during `make javascript`: ulimit -n 10240
 	@rm -rf dev/lib
-	@dev/run -n 1 -q --with-admin-party-please test/javascript/run $(suites)
+	@dev/run -n 1 -q --with-admin-party-please \
+            -c 'startup_jitter=0' \
```

Comment:
This is to make the javascript test pass in Travis. Without this change the integration test suite takes too long and times out. There are enough replications starting that 5 second average wait starts to add up quickly.